apps/searchjoin/db.h

```diff
@@ -130,25 +130,25 @@ namespace seqan
 template <>
 struct Fibre<TDbDnaSaSmall, FibreSA>
 {
-    typedef String<Pair<unsigned int, unsigned char, BitPacked<24, 8> >, DefaultIndexStringSpec<TDbDnaSaSmall>::Type> Type;
+    typedef String<Pair<unsigned int, unsigned char, BitPacked<24, 8> >, StringSpec<TDbDnaSaSmall>::Type> Type;
```

Comment:
```DefaultIndexStringSpec``` is marked deprecated in favor of StringSpec here: https://github.com/seqan/seqan/blob/master/include/seqan/index/index_base.h#L197
so I replaced it by StringSpec (and added the deprecation to the documentation).